tests/PHPUnit/Integration/WidgetsListTest.php

```diff
@@ -37,7 +37,7 @@ public function testGet()
         // check if each category has the right number of widgets
         $numberOfWidgets = array(
             'VisitsSummary_VisitsSummary'  => 6,
-            'Live!'                        => 4,
+            'Live!'                        => 5,
```

Comment:
Please also add the widget info to https://github.com/piwik/piwik/blob/2.16.0-rc1/plugins/Widgetize/tests/System/WidgetTest.php#L829 . This test should automatically notice it is missing but noticed it is not noticing it automatically